### PR TITLE
Feature/93 nested tabs

### LINKED ISSUE #93: Nested Tabs Block Won't Switch Tabs
If you have a tabs component inside another tabs component you aren't able to interact with the tabs of the inner component: https://www.screencast.com/t/jqzcKFaZ

### PR DESCRIPTION
**Pull request by xavortm**
### Description of the Change
Add support for nested tabs; Also add it in the Demo, though I am not sure if it should be there ... I was thinking also that this should get some tests added? Should this happen now or there would be a separate ticket for adding more tests to the tabs?

Screenshot 2021-12-22 at 14 58 50

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by nicholasio** [comment]
Thanks for your PR @xavortm. It would be great if you could add some tests to this. I think e2e (cypress) tests are enough (and easier to write).

Let me know if you need any help.

**Comment by xavortm** [comment]
Thank you for the note, I didn't know before how to write these tests, so I am happy to have pushed the first ^^ Please let me know if there is any problem to the setup though

**Comment by s3rgiosan** [comment]
@nicholasio is there any way of making this a priority? We are using this component on a client, and they have this requirement.

**Review by nicholasio — APPROVED** [review]
LGTM.

@devinle Can you take a look as well/

File: packages/tabs/src/tabs.js [hunk 1]
```diff
@@ -161,8 +161,9 @@ export default class Tabs {
 	 * @returns {Array}
 	 */
 	getTabLinksAndList(tabArea) {
-		const tabLinks = tabArea.querySelectorAll('.tab-list [role="tab"]');
-		const tabList = tabArea.querySelector('.tab-list');
+		const tabParent = tabArea.querySelector('.tab-control');
```
Comment (devinle) [inline packages/tabs/src/tabs.js:164]: @xavortm do we want to check for tabParent existence here?

Comment (devinle) [reply packages/tabs/src/tabs.js:164]: Likely want to check for all of these selectors' existence where needed.

Comment (devinle) [reply packages/tabs/src/tabs.js:164]: As per our conversation in Slack, this is a larger refactor for the safety checks. I think you've resolved the issue at hand. Closing for now.

**Review by devinle — APPROVED** [review]

**Pull request merged**